src/components/button-toggle/README.md

```diff
@@ -0,0 +1,97 @@
+# MdButtonToggle
+
+`MdButtonToggle` is a group of buttons that can be toggled. There are two modes, `multiple` and
+`exclusive`. `Multiple` is a group in which multiple buttons can be toggled at once. `Exclusive` is
```

Comment:
I think it might be clearer if you say "selected" here, rather than "toggled".  Technically all of them can be toggled in any mode, but only one (or multiple) can be selected.